### PR TITLE
T84 admin html to php

### PR DESCRIPTION
**Pull request by CCornez**
made a new structure with admin.php. I changed the vite config as well so that we can use JavaScript in admin.php

### CODE REVIEW

File: php/admin.php [hunk 1]
```diff
@@ -0,0 +1,322 @@
+<?php
+$manifest = file_get_contents("../dist/manifest.json");
+$manifestObject = json_decode($manifest, true);
+?>
```
Comment (CCornez) [inline php/admin.php:4]: Only 3 changes. This is the first change

File: php/admin.php [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Admin Page</title>
+  <link rel="stylesheet" href="../dist/<?= $manifestObject["js/admin.css"]["file"] ?>">
+  <script src="../dist/<?= $manifestObject["js/admin.js"]["file"] ?>" defer></script>
```
Comment (CCornez) [inline php/admin.php:15]: change 2

File: php/admin.php [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+</head>
+
+<body>
+  <!-- admin page start -->
+  <div class="navigation">
+    <ul>
+      <li class="list">
+        <b></b>
+        <b></b>
+        <a href="../index.php">
```
Comment (CCornez) [inline php/admin.php:25]: change 3

Comment (CCornez) [inline php/admin.php:25]: change 3

**Comment by CCornez** [comment]
[admin html to php](https://trello.com/c/ks5QyV4I/84-admin-html-to-php)

File: php/admin.php (hunk 3 above)
Comment (CCornez) [inline php/admin.php:25]: TEST

**Review by BirhanBey — APPROVED** [review]
very nice construction man. congrats 👏

**Pull request merged**